media/css/products/vpn/landing.scss

```diff
@@ -0,0 +1,424 @@
+        .vpn-content-well-image-wrapper {
+            margin-top: 0;
+            padding: ($spacing-xl * 2) $spacing-2xl;
+            width: calc(50% - (#{$spacing-2xl} * 2));
+        }
+
+        &:nth-child(odd) {
+            &:after {
+                left: $spacing-xl;
+                top: $spacing-xl;
+            }
+        }
+
+        &.vpn-content-well-left-half {
+            .vpn-content-well-copy {
+                @include bidi((
+                    (float, left, right),
+                ));
+            }
+
+            .vpn-content-well-image-wrapper {
+                @include bidi((
+                    (float, right, left),
+                ));
+            }
+        }
+
+        &.vpn-content-well-right-half {
+            .vpn-content-well-copy {
+                @include bidi((
+                    (float, right, left),
+                ));
+            }
+
+            .vpn-content-well-image-wrapper {
+                @include bidi((
+                    (float, left, right),
+                ));
+            }
+        }
+    }
+
+    @supports(display: grid) {
+        overflow: visible;
+
+        @media #{$mq-md} {
+            @include grid-column-gap($spacing-xl);
+            display: grid;
+
+            .vpn-content-well-copy {
+                @include flexbox;
+                align-items: center;
+            }
+
+            &.vpn-content-well-left-half {
+                .vpn-content-well-copy {
+                    grid-column: 1;
+                    grid-row: 1;
+                    width: auto;
+                }
+
+                .vpn-content-well-image-wrapper {
+                    grid-column: 2;
+                    grid-row: 1;
+                    width: auto;
+                }
+            }
+
+            &.vpn-content-well-left-half {
+                grid-template-columns: 1fr 1fr;
+            }
+
+            &.vpn-content-well-right-half {
+                .vpn-content-well-copy {
+                    grid-column: 2;
+                    grid-row: 1;
+                    width: auto;
+                }
+
+                .vpn-content-well-image-wrapper {
+                    grid-column: 1;
+                    grid-row: 1;
+                    width: auto;
+                }
+            }
+
+            &.vpn-content-well-right-half {
+                grid-template-columns: 1fr 1fr;
+            }
+
+        }
+
+        @media #{$mq-lg} {
+            @include grid-column-gap($spacing-2xl);
+        }
+    }
+}
+
+//* -------------------------------------------------------------------------- */
+// FAQ
+
+.vpn-faq {
+    border-top: 4px solid $color-black;
+    padding-top: $layout-xl;
+}
+
+.vpn-faq-section-title {
+    @include text-title-md;
+    margin-bottom: $spacing-2xl;
+    text-align: center;
+}
+
+.vpn-faq-item-title {
+    background: $color-marketing-gray-20;
+    box-shadow: 8px 8px 0 $color-black;
+    margin-bottom: $spacing-lg;
+    padding: $spacing-lg;
```

Comment:
The spacing in the open item is much larger than vpn.moz. It's about ~40px. I recommend loosing the bottom margin here. The padding around the `.vpn-faq-item-title` and `.vpn-faq-item-detail` is enough. 

```suggestion
    padding: $spacing-lg;
```

vpn.moz.org: 
<img width="758" alt="image" src="https://user-images.githubusercontent.com/2692333/108173571-eb16b680-70c3-11eb-8913-3a30919cc8ad.png">

moz.org/vpn: 
<img width="755" alt="image" src="https://user-images.githubusercontent.com/2692333/108173586-eeaa3d80-70c3-11eb-8d8f-13c7f7cda949.png">

Reply:
Good call 👍